.circleci/config.yml

```diff
@@ -22,13 +22,13 @@ jobs:
       - restore_cache:
           name: Restore PNPM Package Cache
           keys:
-            - v3-maskbook-cache-pnpm
+            - v3-maskbook-cache-pnpm-{{ checksum "pnpm-lock.yaml" }}
```

Comment:
pnpm has a "global" store and a "local store". You can think of "local store" as a hard link to the "global store". And here we're catching the global store (cache) not the local store (node_modules) so I think we should not add a checksum to the package-lock.

Reply:
I see.